packages/react-components/src/AccessibilityScenarios/Input.stories.tsx

```diff
@@ -0,0 +1,209 @@
+import * as React from 'react';
+
+import { Input, InputOnChangeData } from '@fluentui/react-input';
+import { Label } from '@fluentui/react-label';
+import { Button } from '@fluentui/react-button';
+
+import { Scenario } from './utils';
+
+export const RegistrationFormInputsAccessibilityScenario = () => {
```

Comment:
Rather than writing custom validation, could we make this an example of integrating a common form validation library? The rest of the comments assume that we keep the custom validation approach, but overall I think demonstrating an existing library would be better.